Exygy/askdarcel-web
# Footer

# Pull Request Template

## Issue Overview

This PR addresses [#91](https://www.notion.so/exygy/Footer-5b7c37dec020453db5601aae545c4ff5)
CMS PR (Sanity schema) here: https://github.com/Exygy/our415-cms/pull/4


## Description

This PR replaces the old footer with the new mockup version, as well as pulls the contact info, column titles and links from Sanity Studio. 

## How Can This Be Tested/Reviewed?

Add sanity env vars from `config.example.yml` into `config.yml` and add the project id var from sanity studio settings.

To review, visit any page and scroll down to the footer. 

Visual / A11y QA: 
- Should be correctly populated from Sanity Studio
- Internal links should open in the same tab (starting with '/'). External links should open in a different tab (all links not starting with '/'. There are a few placeholders with href '#').
- View on different sized screens
- Tab and screen reader navigate through links 

Sanity:
- You can edit the footer in the Our415 Development sanity project and the changes should reflect in the footer

No tests yet.

## Checklist:


## Reviewer Notes:

Steps to review a PR:

- Read and understand the issue, and ensure the author has added QA notes
- Review the code itself from a style point of view
- Pull the changes down locally and test that the acceptance criteria is met
- Also review the acceptance criteria on the Netlify deploy preview (noting that these do not yet include any backend changes made in the PR)
- Either explicitly ask a clarifying question, request changes, or approve the PR if there are small remaining changes but the PR is otherwise good to go

## On Merge:

If you have one commit and message, squash. If you need each message to be applied, rebase and merge.
